package.json

```diff
@@ -344,7 +344,11 @@
     "browserify-sign": "^4.2.3",
     "follow-redirects": "^1.15.6",
     "redux": "^5.0.1",
-    "braces": "^3.0.3"
+    "braces": "^3.0.3",
+    "ws@5.0.0": "^5.2.4",
+    "ws@6.0.0": "^6.2.3",
+    "ws@7.0.0": "^7.5.10",
+    "ws@8.0.0": "^8.17.1"
```

Comment:
I think if you'd need to use ws@5.x, ws@6.x, etc. This seems like it would resolve only exact `5.0.0`, `6.0.0` versions

Reply:
That didn't work either. Based on @jh2oman response in [Slack](https://valora-app.slack.com/archives/C02898GN22V/p1718663436724709?thread_ts=1718660096.201749&cid=C02898GN22V); I've opted to ignore the vulnerability.